### PR TITLE
[DO NOT MERGE BEFORE TP2] Creating a new development environment

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: hooks/pre_validations/10-gather_and_set_fusor_values.rb [hunk 1]
```diff
@@ -46,8 +46,8 @@
   param('capsule', 'bmc_default_provider').value = provisioning_wizard.bmc_default_provider
   param('certs', 'node_fqdn').value = provisioning_wizard.fqdn
   param('certs', 'ca_common_name').value = provisioning_wizard.fqdn
-  param('foreman', 'servername').value = provisioning_wizard.fqdn
-  param('foreman', 'foreman_url').value = provisioning_wizard.base_url
+  #param('foreman', 'servername').value = provisioning_wizard.fqdn
```
Comment (daviddavis) [inline hooks/pre_validations/10-gather_and_set_fusor_values.rb]: These don't seem to work in development mode.

File: hooks/pre_values/10-register_fusor_modules.rb [hunk 2]
```diff
@@ -4,5 +4,5 @@
              {:manifest_name => 'plugin/fusor',  :dir_name => 'foreman'})
 
   # make sure foreman-tasks is enabled
-  kafo.module('foreman_plugin_tasks').enable
+  #kafo.module('foreman_plugin_tasks').enable
```
Comment (daviddavis) [inline hooks/pre_values/10-register_fusor_modules.rb]: Figure out why this doesn't work. It doesn't seem to be needed so maybe disable for devel installer?

File: config/fusor-devel-installer.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - certs
+    - katello_devel
+  :password: l7VtnWiPeKe412o2CVBM6yVbTkKGh6L_CKx4_zBkmUE
+  :provisioning_wizard: interactive
+  :dont_save_answers: false
+  :ignore_undocumented: false
+  :interactive: 
+  :noop: false
+  :verbose: 
+  :verbose_log_level: info
```
Comment (jmrodri) [inline config/fusor-devel-installer.yaml:36]: There are 4 lines with trailing whitespaces: 7, 14, 26, 33, 35

Comment (daviddavis) [inline config/fusor-devel-installer.yaml]: Nope, there are 8 lines: 1, 7, 8, 11, 14, 26, 33, 35. :wink:

**Comment by jwmatthews** [comment]
ACK

**Pull request merged**